components/Calendars.js

```diff
@@ -1107,9 +1122,9 @@ class ReservationDayEventBase extends Component {
                             return (
 
                                 <TouchableOpacity key={event?.id} onPress={() => {
-                                    if (event.status !== 'CANCELLED') {
+                                    if (event.status !== 'CANCELLED' && event.status !== 'CONFIRMED') {
```

Comment:
What is this condition check for?

Reply:
I think if it's confirmed, the reservation can't change.